Packs/FeedProofpoint/Integrations/FeedProofpoint/FeedProofpoint.py

```diff
@@ -87,7 +87,12 @@ def _build_iterator(
                 # add type/value to item.
                 if "domain" in item:
                     item["type"] = FeedIndicatorType.Domain
-                    item["value"] = item.get("domain")
+                    # As part of the domain feed, also DomainGlob indicators will be returned, so we are checking if the
+                    # domain has '*' in their value
+                    indicator_value = str(item.get("domain", ""))
```

Comment:
Why are you calling `str` here?

Reply:
For some reason running line 93 - `if '*' in indicator_value:` resulted in a type error was solved by adding this casting. I Could change the str casting to be inside the if statement like this: `if '*' in str(indicator_value):`

Reply:
I suppose that could happen if the value for that key is explicitly `None` but if that were the case, then we shouldn't even be yielding an indicator object for that iteration. Please investigate why that may be happening. If there are iterations for which the key `domain` is present but the value is `None`, we should be filtering those out.